kolibri/core/discovery/utils/filesystem/windows.py

```diff
@@ -69,7 +69,7 @@ def _wmic_output():
     )
 
     # pipe output from the WMIC command to the temp file
-    cmd = "wmic logicaldisk list full /format:csv > {}".format(OUTPUT_PATH)
+    cmd = "wmic logicaldisk list full /format:\"%WINDIR%\\System32\\wbem\\en-us\\csv\" > {}".format(OUTPUT_PATH)
```

Comment:
This seems like the right fix, and I can see by googling that people are also having issues because a `csv.xls` isn't even present in some activated non-`en-US` locale.

If necessary, you could check if the `csv.xls` file exists before using it? I can't really find a specification about which Windows versions this file is present on?

[This post](https://twitter.com/3gstudent/status/988934413554614273) implies that by specifying just `csv`, then any `csv.xls` in the current path will be used. It might be an okay fallback.

Reply:
yes it turns out that this command doesn't work on Windows XP, since the `csv.xls` only exists under `%WINDIR%\System32\wbem` there. Do you think it would be okay for me to check if `csv.xls` exists in `%WINDIR%\System32\wbem\en-us`, and if not, then fall back to ``%WINDIR%\System32\wbem`? Thank you!

Reply:
...yes, and maybe, finally falling back to the old `csv` ? :)

Reply:
ah yes! that sounds better! thank you for the suggestion.